node-memory-considerate-loading/server.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const express = require('express');
+// const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const request = require('request');
+
+const app = express();
+app.disable('x-powered-by');
+app.use(cors());
+
+app.use(express.static(path.join(__dirname, 'public')));
+app.set('views', __dirname + '/public');
+app.engine('html', require('ejs').renderFile);
+app.set('view engine', 'html');
+
+app.get('/ping', (req, res) => {
+  res.send('pong');
+});
+
+app.get('/memory-considerate-image', (req, res) => {
+  // TODO: As this is a demo, I think it should be easy enough to change these numbers as needed in the future.
+  const MEMORY_LIMIT = 4;
+  const deviceMemory = req.headers['device-memory'];
```

Comment:
Could we add a comment above this line that includes a link back to https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/client-hints/#device_hints?

Reply:
`// inspired by https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/client-hints/#device_hints`

I added like above.